lib/features/mailbox/presentation/mixin/mailbox_widget_mixin.dart

```diff
@@ -31,20 +31,24 @@ mixin MailboxWidgetMixin {
     PresentationMailbox mailbox,
     bool spamReportEnabled
   ) {
+    final _notSupportShowMarkAsRead = mailbox.isDrafts || mailbox.isSent || mailbox.isTrash;
```

Comment:
IMO, we not need it, this condition is only to show the number of unread messages